packages/react-dev-utils/paths/printFileSizes.js

```diff
@@ -7,6 +7,11 @@ var gzipSize = require('gzip-size').sync;
 
 // Print a detailed summary of build files.
 module.exports = paths => function printFileSizes(stats, previousSizeMap) {
+  function removeFileNameHash(fileName) {
```

Comment:
ah, I reverted this because removeFileNameHash is still required by build.js line 52